### DISCUSSION TITLE
TheTVDB Lookup plugin update for new api access

### DISCUSSION THREAD

**Discussion by bpepple** [Flexget/Flexget#2875] [Q&A]
Has there been any work done to the TheTVDB Lookup pluigin for the upcoming access changes to the thetvdb.com api? Looked around a bit and couldn't find any references.

**Comment by ksurl** [comment]
I don't think any active work is being done to support the changes. Tmdb or tvmaze, etc may be your best bet if you need it

**Comment by frankyw** [comment]
Is flexget included in the list of apps that will require a key, I noted there were some exceptions but could not find any detail.

Has the change kicked in yet, I have not seen any changes in usage so far?

**Reply by bpepple** [reply]
I don't believe TVDB are making exceptions, but some some large users (Plex) are purchasing an API key for their app.  Most open source projects aren't able to afford to do that, so they will need their users to purchase an individual key to use the TVDB api.

**Comment by thekevmp** [comment]
is there a way we can enter in our own tvdb api key into flexget?

**Reply by crawln45** [reply]
From what I remember not currently unless you edit your local files. We don't have a way of injecting your key that I can remember.

**Reply by thekevmp** [reply]
thank you for the reply, not sure if I have the tech savvy to look into it.

currently is there a way to search for a series by using it's tvmaze or tvdb ID?  I checked the cookbook and wasn't sure exactly how to integrate the options.

**Reply by crawln45** [reply]
Tvdbid or IMDb Id is the best way to search meta data.
